app/src/main/res/layout/activity_payment.xml

```diff
@@ -134,7 +158,9 @@
                 android:layout_height="wrap_content"
                 android:layout_weight="0" />
             <TextView
-                android:text="Lưu ý "
+                android:ellipsize="end"
```

Comment:
[Android, How to limit width of TextView (and add three dots at the end of text)